core/trino-main/src/main/java/io/trino/execution/executor/TaskExecutor.java

```diff
@@ -307,6 +315,13 @@ private void doRemoveTask(TaskHandle taskHandle)
         log.debug("Task finished or failed %s", taskHandle.getTaskId());
     }
 
+    private void recordAllSplitQueueSize()
```

Comment:
synchronized this explicitly

`recordAllSplitQueueSize` -> `recordAllSplitsQueueSize`